### ISSUE TITLE
one-to-one elink requests with NCBI Entrez

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#361]
There is a subtle difference in the elink results for multiple identifiers, _Link from protein to gene_ http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=protein&db=gene&id=15718680,157427902,119703751 versus _Find one-to-one links from protein to gene_ http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=protein&db=gene&id=15718680&id=157427902&id=119703751 on http://www.ncbi.nlm.nih.gov/books/NBK25499/

Currently the way we build the URL gives the first form only. See https://www.biostars.org/p/112225/

Update: See BioStars, we need to do check both `elink` and `epost`

**Comment by erasche** [comment]
Oh...Oh goodness. @peterjc any thoughts of how you'd like to see this ... special behaviour reflected in the Galaxy wrappers?

**Comment by peterjc** [comment]
@erasche not really, no. Very tricky to present a nice end-user interface for this.

Does the argument type dependant behaviour outlined here seem like a sane idea?

**Comment by erasche** [comment]
@peterjc it seems like the best possible option.

**Comment by peterjc** [comment]
@mdehoon I think we just need to document this subtly in the docstring, and then we can close this issue...

**Comment by peterjc** [comment]
Other than documenting it, ``elink`` looks OK. Time to do the same for ``epost``?

**Comment by erasche** [comment]
Does `epost` really behave differently? I'm surprised. Oh Saw your edit, nvm.

**Comment by peterjc** [comment]
Sorry - confused myself about ``elink`` vs ``epost``. Hopefully clear now - the BioStars query certainly suggests both need looking at.
